### PR TITLE
Feature/x platform

### PR DESCRIPTION
**Pull request by shanejonas**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,8 +5,8 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "clean": "rm -rf emerald-tool && rm -rf emerald-wallet && rm -f svmdev",
-    "postinstall": "export APP_VERSION_GIT_TAG=v1.0.0-beta-emerald-cli && npm run clean && node downloaders/sputnikvm-dev.js && node downloaders/emerald-wallet.js && node downloaders/emerald-explorer"
+    "clean:deps": "rimraf emerald-explorer EmeraldWallet.app EmeraldWallet.dmg svmdev EmeraldWallet.AppImage EmeraldWallet.exe",
+    "postinstall": "npm run clean:deps && node downloaders/sputnikvm-dev.js && node downloaders/emerald-wallet.js && node downloaders/emerald-explorer"
```
Comment (BelfordZ) [inline package.json]: bump package version

File: downloaders/emerald-wallet.js [hunk 2]
```diff
@@ -1,25 +1,50 @@
...
+            if (err) {
+              return spinner.fail(`An error occurred: ${JSON.stringify(err)}`);
+            }
+            spinner.succeed('emerald-wallet: finished installation');
+          });
+        });
+      });
+  case 'linux':
+    const linuxFile = fs.createWriteStream(path.resolve(rootPath, 'EmeraldWallet.AppImage'));
+    return request('http://builds.etcdevteam.com/emerald-wallet/v1.0.x/EmeraldWallet-linux-x86_64-v1.0.0+53-24bb0c9.AppImage')
```
Comment (BelfordZ) [inline downloaders/emerald-wallet.js]: different wallet version

File: downloaders/emerald-wallet.js [hunk 3]
```diff
@@ -1,25 +1,50 @@
...
+      .pipe(linuxFile)
+      .on('finish', () => {
+        shell.exec(`chmod u+x ${path.resolve(rootPath, 'EmeraldWallet.AppImage')}`);
+        spinner.succeed('emerald-wallet: finished installation');
+      });
+  case 'win32':
+    const windowsFile = fs.createWriteStream(path.resolve(rootPath, 'EmeraldWallet.exe'));
+    return request('http://builds.etcdevteam.com/emerald-wallet/v1.0.x/EmeraldWallet-win-v1.0.0+53-24bb0c9.exe').pipe(windowsFile).on('finish', () => {
```
Comment (BelfordZ) [inline downloaders/emerald-wallet.js]: different wallet version

File: downloaders/sputnikvm-dev.js [hunk 4]
```diff
@@ -2,16 +2,23 @@ const request = require('request');
...
-    case 'darwin': 
-      return request('http://builds.etcdevteam.com/sputnikvm-dev/v0.2.x/svmdev-osx-v0.2.0-36-46a63e0.tar.gz').pipe(extract);
-    case 'linux': 
-      return request('http://builds.etcdevteam.com/sputnikvm-dev/v0.2.x/svmdev-linux-v0.2.0-36-46a63e0.tar.gz').pipe(extract);
+  case 'darwin': 
+    return request('http://builds.etcdevteam.com/sputnikvm-dev/v0.2.x/svmdev-osx-v0.2.0-36-46a63e0.tar.gz').pipe(extract).on('end', () => spinner.succeed('sputnikvm-dev: finished installation'));
+  case 'linux': 
+    return request('http://builds.etcdevteam.com/sputnikvm-dev/v0.2.x/svmdev-linux-v0.2.0-36-46a63e0.tar.gz').pipe(extract).on('end', () => spinner.succeed('sputnikvm-dev: finished installation'));
+  case 'win32': 
+    return request('https://builds.etcdevteam.com/sputnikvm-dev/v0.2.x/sputnikvm-dev-v0.2.0-26-bd7111b-i686-pc-windows-msvc.zip').pipe(unzip.Extract({ path: path.resolve(__dirname, '../') })).on('end', () => spinner.succeed('sputnikvm-dev: finished installation'));
```
Comment (BelfordZ) [inline downloaders/sputnikvm-dev.js]: `v0.2.0-26` is that right?

**Comment by BelfordZ** [comment]
merge after fixes!

**Comment by BelfordZ** [comment]
my bad ^^

File: downloaders/sputnikvm-dev.js (hunk 4 above)
Comment (shanejonas) [reply downloaders/sputnikvm-dev.js]: yep

Comment (shanejonas) [reply downloaders/sputnikvm-dev.js]: need to make get another sputnikvm-dev windows build goin

**Pull request merged**